### PR TITLE
WIP: Chinese translation

### PR DESCRIPTION
**Pull request by phk422**
I have modified the organization of relevant configuration files, please confirm whether this is feasible? If so, I can start translating

### CODE REVIEW

File: docs/.vitepress/zh.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { defineConfig } from 'vitepress'
+
+export const zh = defineConfig({
```
Comment (drwpow) [inline docs/.vitepress/zh.ts:3]: Ah this is smart! I wasn’t thinking about all the additional links and navigation items that needed to be translated.

File: docs/.vitepress/en.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            },
+            { text: "Testing", link: "/openapi-fetch/testing" },
+            { text: "Examples", link: "/openapi-fetch/examples" },
+            { text: "API", link: "/openapi-fetch/api" },
+            { text: "About", link: "/openapi-fetch/about" },
+          ],
+        },
+      ],
+      // 🌐
+      "/zh/": [
```
Comment (drwpow) [inline docs/.vitepress/en.ts:44]: We can probably remove `zh` from the English sidebar, right?

**Review by drwpow — APPROVED** [review]
Looks great!

File: docs/.vitepress/en.ts (hunk 2 above)
Comment (phk422) [reply docs/.vitepress/en.ts]: Yes! I've removed it.

**Comment by drwpow** [comment]
This is looking great! If you’re OK with it, I’m going to merge this, and deploy it. We can add additional pages in followup PRs.

**Pull request merged**